napari/_qt/qt_viewer.py

```diff
@@ -54,6 +56,8 @@ def __init__(self, viewer):
         self.buttons = QtLayersButtons(self.viewer)
         self.console = QtConsole({'viewer': self.viewer})
 
+        self.vispy_layers = {}
```

Comment:
Can you add a comment here about what this dictionary is meant to hold? I'm not sure about `vispy_layers`, either... Maybe `layer_to_visuals`, `layer_visual`, `layer_visual_nodes`, `layer_scenegraph`...?

Reply:
renamed `layer_to_visual` and comment added